### PR TITLE
edits to n-3 PUFA section

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Hey @RLordan! As I was reading, I had some questions about n-3 PUFA + inflammation, so I decided to look into it and write it up in case it's useful info to potentially include! Mainly I am extremely interested in the intersection between inflammation and COVID-19, which we touch on in a few different papers, so I thought this might be a change to build that theme out a little more in this paper.

You are absolutely the expert on this so I'm curious to hear if you think there's anything worth including here! 

Additionally, I'm not sure if omega-3s and n-3 PUFA are exactly the same thing, or whether there's just overlap, so I tried to stick with the terminology used in the paper -- please feel free to adjust based on your understanding!

I also have some superficial edits for the other sections, but figured I should post this separately since it's a little more involved.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/21.nutraceuticals.md [hunk 1]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
+However, they also reported that most of their tests were not statistically significant, and therefore it seems that additional research using larger sample sizes is required.
+A recent meta-analysis of 10 randomized controlled trials (RCTs) examining the effects of omega-3 fatty acids on ARDS patients did not find evidence of any effect on mortality, although the effect on secondary outcomes could not be determined due to a low quality of evidence [@doi:10.4187/respcare.06965].
+However, another meta-analysis that examined 24 RCTs studying the effects of omega-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when omega-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].
+Therefore, despite theoretical support for an immunomodulatory effect of n-3 PUFAs in COVID-19, evidence from existing RCTs is insufficient to determine whether supplementation offers an advantage in a clinical setting that would be relevant to COVID-19.
```
Comment (RLordan) [inline content/21.nutraceuticals.md:98]: I really like this section, good on you @rando2!

File: content/21.nutraceuticals.md [hunk 2]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
-As a result, the use of other therapeutics, such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities, may be beneficial and warrant further investigation.
+In particular, concerns have been raised within the medical community about using investigational therapeutics on COVID-19 patients who are already on antiplatelet therapies due to pre-existing comorbidities because the introduction of such therapeutics could lead to issues with dosing and drug choice and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
+In such cases, dietary sources of n-3 fatty acids or other nutraceuticals with antiplatelet activities could hold particular value for reducing the risk of thrombotic complications in patients already receiving pharmaceutical antiplatelet therapies. <!--TO DO: is this being explored/tested?-->
```
Comment (RLordan) [inline content/21.nutraceuticals.md:117]: I would say that this is being explored in the literature, but I haven't seen any evidence of trials looking at platelet activities as a primary or secondary endpoint. Its a labour intensive trial if looking beyond the standard prothrombin time etc.

Comment (RLordan) [reply content/21.nutraceuticals.md:117]: As you see below, they are mainly interested in the anti-inflammatory potential.

File: content/21.nutraceuticals.md [hunk 3]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
+Interest has also arisen as to whether nutritional status related to n-3 PUFAs can also affect inflammation associated with severe disease, such as ARDS or sepsis [@doi:10.1007/s11745-016-4129-x; @doi:10.1016/j.nut.2019.02.016].
+ARDS and sepsis hold particular concern in the treatment of severe COVID-19;  an analysis of 82 deceased COVID-19 patients in Wuhan during January to February 2020 reported that respiratory failure (associated with ARDS) was the cause of death in 69.5% of cases, and sepsis or multi-organ failure accounted for 28.0% of deaths [@doi:10.1371/journal.pone.0235458].
+Research in ARDS prior to current pandemic suggests that n-3 PUFAs may hold some therapeutic potential.
+One study randomized 16 consecutive ARDS patients to receive either an FO-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].
+They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the FO-enriched emulsion, suggesting that the FO supplementation may have reduced inflammation.
```
Comment (RLordan) [inline content/21.nutraceuticals.md:94]:
```suggestion
They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the fish oil-enriched emulsion, suggesting that the FO supplementation may have reduced inflammation.
```

File: content/21.nutraceuticals.md [hunk 4]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
+However, they also reported that most of their tests were not statistically significant, and therefore it seems that additional research using larger sample sizes is required.
+A recent meta-analysis of 10 randomized controlled trials (RCTs) examining the effects of omega-3 fatty acids on ARDS patients did not find evidence of any effect on mortality, although the effect on secondary outcomes could not be determined due to a low quality of evidence [@doi:10.4187/respcare.06965].
+However, another meta-analysis that examined 24 RCTs studying the effects of omega-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when omega-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].
```
Comment (RLordan) [inline content/21.nutraceuticals.md:97]:
```suggestion
However, another meta-analysis that examined 24 RCTs studying the effects of n-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when n-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].
```

File: content/21.nutraceuticals.md [hunk 5]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
-Another potential mechanism that has led to interest in n-3 PUFA for viral infections is its potential as a precursor molecule for the biosynthesis of endogenous specialized proresolving mediators (SPM), such as protectins and resolvins, that actively resolve inflammation and infection [@doi:10.1016/j.immuni.2014.02.009].
-Finally, some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis [@doi:10.1182/blood.2020006000; @doi:10.1007/s11239-020-02134-3].
-Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration [@doi:10.1016/j.jacc.2020.04.031; @doi:10.3389/fnut.2020.583080], which could potentially include n-3 PUFA.
-
+One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
```
Comment (RLordan) [inline content/21.nutraceuticals.md:80]:
```suggestion
One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], commonly referred to as omega-3 fatty acids, which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
```

File: content/21.nutraceuticals.md [hunk 6]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
+Interest has also arisen as to whether nutritional status related to n-3 PUFAs can also affect inflammation associated with severe disease, such as ARDS or sepsis [@doi:10.1007/s11745-016-4129-x; @doi:10.1016/j.nut.2019.02.016].
+ARDS and sepsis hold particular concern in the treatment of severe COVID-19;  an analysis of 82 deceased COVID-19 patients in Wuhan during January to February 2020 reported that respiratory failure (associated with ARDS) was the cause of death in 69.5% of cases, and sepsis or multi-organ failure accounted for 28.0% of deaths [@doi:10.1371/journal.pone.0235458].
+Research in ARDS prior to current pandemic suggests that n-3 PUFAs may hold some therapeutic potential.
+One study randomized 16 consecutive ARDS patients to receive either an FO-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].
```
Comment (RLordan) [inline content/21.nutraceuticals.md:93]:
```suggestion
One study randomized 16 consecutive ARDS patients to receive either a fish oil-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].
```

**Review by RLordan — APPROVED** [review]
Hey @rando2, I love these additions. The inflammation/n-3 fatty acid story has really been exciting over the last decade! Thanks for adding to it :) N-3 Pufa and omega-3 are the exact same thing. Same story for n-6 PUFA and omega-6. It just refers to the carbon that the first double-bond in the molecule appears e.g. carbon 3 or 6. It seems this has many biological consequences. n-3 are mostly thought to be anti-inflammatory and n-6 mostly pro-inflammatory, but as you know, nothing is ever that clear! haha 

I added a bit to the first line to highlight that and just changed everything to n-3 PUFA so that we are consistent.

I'm thinking I might have one or two articles that may be useful to cite here later today. I am just finishing off the variants of concern section for evolution at the moment, I might add that tonight for your review. But your additions are good to go, just a minor change suggestion. Thanks for all your help with this :)

**Comment by rando2** [comment]
Thank you so much @RLordan!

File: content/21.nutraceuticals.md [hunk 7]
```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
-Finally, some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis [@doi:10.1182/blood.2020006000; @doi:10.1007/s11239-020-02134-3].
-Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration [@doi:10.1016/j.jacc.2020.04.031; @doi:10.3389/fnut.2020.583080], which could potentially include n-3 PUFA.
-
+One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], commonly referred to as omega-3 fatty acids, which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
+EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils (FO) or purified oils [@doi:10.1039/c9fo01742a].
```
Comment (rando2) [inline content/21.nutraceuticals.md:81]:
```suggestion
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils [@doi:10.1039/c9fo01742a].
```

**Comment by rando2** [comment]
Failed because rate limit exceeded, so merging off of manubot pass

**Pull request merged**